js/stores/appStore.js

```diff
@@ -255,7 +255,10 @@ class AppStore extends EventEmitter {
 
   emitChanges (emitFullState) {
     if (lastEmittedState) {
-      const d = diff(lastEmittedState, appState)
+      // diff and patch will make map out of order so toList() is to maintain
+      // sites map order
+      const diffAppState = appState.set('sites', appState.get('sites').toList())
```

Comment:
We need a better way to deal with the our of order.
This will cause significant slowness